src/status_im/contexts/chat/messenger/messages/drawers/view.cljs

```diff
@@ -93,6 +93,11 @@
       (rf/dispatch [:pin-message/send-pin-message
                     (assoc message-data :pinned message-not-pinned?)]))))
 
+(defn message-pin-shown?
+  [community?]
+  (let [able-to-send-message? (rf/sub [:chats/able-to-send-message?])]
```

Comment:
A minor detail, but usually we try to subscribe in view components and then pass the results to pure functions. One reason is because subscriptions are really tied to the component life cycle and when we put them inside functions it can give the wrong idea that the function can be called from anywhere.